src/modals/Validate.tsx

```diff
@@ -109,37 +101,32 @@ function Validate(props: ValidateProps) {
 
     const createSmartWallet = async () => {
         if (state.provider) {
-            dispatch({ type: 'set_loader', loader: true });
+            // dispatch({ type: 'set_loader', show: true });
```

Comment:
Is it still necessary? If not I'd remove it.

Reply:
updated